app/server/models/user.py

```diff
@@ -570,34 +570,28 @@ def get_most_relevant_transfer_usage(self):
                 LEFT JOIN "user" u ON c.recipient_user_id = u.id
                 WHERE sender_user_id = {} AND c.transfer_status = 'COMPLETE'
                 ORDER BY c.updated DESC
-                LIMIT 20)
+                )
             C GROUP BY business_usage_id ORDER BY count DESC
         '''.format(self.id))
         result = db.session.execute(sql)
-        transfer_usage_ids = [row[0] for row in result]
+        most_common_ids = [row[0] for row in result]
         # Get most common business_usage_id
-        number_of_wanted_business_id = 9
 
-        most_common_ids = transfer_usage_ids[:number_of_wanted_business_id]
-        # If less than the wanted nr of usage is found fill up the list with default
-        if len(most_common_ids) < number_of_wanted_business_id:
-            most_relevant_ids = self.refine_with_default_transfer_usages(
-                most_common_ids, number_of_wanted_business_id)
+        most_relevant_ids = self.refine_with_default_transfer_usages(
+            most_common_ids)
         relevant_transer_usages = TransferUsage.query.filter(
```

Comment:
is this still necessary? now all TransferUsages are in this list right?

Reply:
No, first the most common Transferusages are listed, so the ones that a user has sent a tranfer too, and than the list is supplemented with the default usages. 

We could however extend that list even further with basically the rest of the tranferusages. Then the list would be in this order:
- The transferusages that the user has used before (can either be default or not default)
- The default transferusage
- All other tranfer 

What do you think? 

For now I'm leaving it as is. Unless you guys think the above mentioned change makes sense (or another change)

Reply:
@RubenGeo You removed the limit on the queries. Aren't we having the user page
through all possible usages now so they'll see all the options ordered by
most used? So default or not are all going to show up in the list